app/Http/Controllers/OrderController.php

```diff
@@ -79,6 +99,14 @@ public function update(Request $request, $id)
      */
     public function destroy($id)
     {
-        //
+        $order = Order::find($id);
+   
+        $order->delete();
+        alert()->warning('d','d');
```

Comment:
```suggestion
        alert()->warning('d', 'd');
```
Mà mấy cái alert này sao toàn có 1 chữ cái là sao nhỉ?